apps/teamsjs-cc-app/tabs/src/helpers/index.ts

```diff
@@ -0,0 +1,9 @@
+export const booleanToString = (value: boolean) => {
```

Comment:
Nitpick: This file is named "index.ts", and that name usually manes something that is specifically exporting things from inside the package for use outside the package. Is there a better filename for this?

Reply:
Updated file name!